test/contract-tests/tests/test/amazon/awssdk/awssdk_test.py

```diff
@@ -363,7 +496,7 @@ def _assert_semantic_conventions_attributes(
         attributes_dict: Dict[str, AnyValue] = self._get_attributes_dict(attributes_list)
         self._assert_str_attribute(attributes_dict, SpanAttributes.RPC_METHOD, operation)
         self._assert_str_attribute(attributes_dict, SpanAttributes.RPC_SYSTEM, "aws-api")
-        self._assert_str_attribute(attributes_dict, SpanAttributes.RPC_SERVICE, service.split("::")[-1])
+        self._assert_str_attribute(attributes_dict, SpanAttributes.RPC_SERVICE, rpc_service)
```

Comment:
why is change required?

Reply:
See the description. We have to manually pass in `rpc_service` for Bedrock since it can't be extracted from the AWS::* service name.

Reply:
Ok. I saw the explanation in overview section. Could you add some comments in code?

Reply:
added.